lib/main_screen.dart

```diff
@@ -29,109 +29,128 @@ class MainScreenState extends State<MainScreen> {
   @override
   Widget build(BuildContext context) {
     GlobalBloc globalBloc = GlobalProvider.of(context);
+    double width = MediaQuery.of(context).size.width;
     return Scaffold(
-      appBar: AppBar(
-        leading: Image.asset(
-          'assets/eventhub-logo2.png',
-          scale: 8,
-        ),
-        title: Text("EventHub"),
-        actions: <Widget>[
-          IconButton(
-            icon: Icon(Icons.info_outline),
-            onPressed: () {
-              showAboutDialog(
-                  context: context,
-                  applicationVersion: "1.0",
-                  applicationIcon: Card(
-                      color: Colors.red[800],
-                      child: Padding(
-                        padding: const EdgeInsets.all(8.0),
-                        child: Image.asset(
-                          'assets/eventhub-logo2.png',
-                          scale: 8,
-                        ),
-                      )),
-                  applicationName: 'EventHub',
-                  children: [
-                    Linkify(
-                      text: '''
+      backgroundColor: Color(0xffB54646),
```

Comment:
You should never hardcode colors like this.
Please use `Theme.of(context).primaryColor` instead

Reply:
Ok. It was hardcoded in splash screen so I just copied it. No problem will see.

Reply:
Some parts of this code is about 2 years old.
So you can change that as well.